src/NHMCore/ApplicationState/VersionState.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using NHM.Common;
+using NHMCore.Notifications;
 using NHMCore.Utils;
 using System;
 using System.Windows.Forms;
```

Comment:
@luc1an24
give option for manual update of miner
ed2d57e
@luc1an24
use dispatcher for action